benches/gamma.rs

```diff
@@ -37,3 +37,52 @@ fn inc_gamma(bencher: &mut Bencher) {
         }
     });
 }
+
+fn trigamma(value: f64) -> f64 {
```

Comment:
Is it an alternative implementation? I don’t think this is needed here. We benchmark what the packages provides.

Reply:
Removed the alternative version and its benchmark